components/CryptoSection.tsx

```diff
@@ -0,0 +1,128 @@
+import { useCallback, useMemo, useState } from "react";
+import Web3 from "web3";
+import { AbiItem } from "web3-utils";
+
+import showToast from "@lib/notification";
+
+import { Button } from "@components/ui/Button";
+
+import genericAbi from "../web3/abis/abiWithGetBalance.json";
+
+interface Window {
+  ethereum: any;
+  web3: Web3;
+}
+
+interface EvtsToVerify {
+  [eventId: string]: boolean;
+}
+
+declare const window: Window;
+
+interface CryptoSectionProps {
+  id: number | string;
+  scAddress: string;
+  oneStep: boolean; // When set to true, there will be only 1 button which will both connect Metamask and verify the user's wallet. Otherwise, it will be in 2 steps with 2 buttons.
```

Comment:
```suggestion
  /** When set to true, there will be only 1 button which will both connect Metamask and verify the user's wallet. Otherwise, it will be in 2 steps with 2 buttons. */
  oneStep: boolean;
```

This will allow IDE to display the comment in IDEs like VS Code (better DX overall)